### ISSUE TITLE
Running make build throws checksum mismatch on gotest.tools@v2.2.0+incompatible

### ISSUE THREAD

**Issue by TarunParimi** [cloudera/yunikorn-k8shim#15]
Running `make build` locally gives the following error.
```
go: verifying gotest.tools@v2.2.0+incompatible: checksum mismatch
downloaded: h1:y0IMTfclpMdsdIbr6uwmJn5/WZ7vFuObxDMdrylFM3A=
go.sum:     h1:VsBPFP1AI068pPrMxtb/S8Zkgf9xEmTLJjfM+P5UIEo=
```

Looks like this occurs when using go version 1.11.4+ https://github.com/golang/go/issues/29278

The trivial way to fix this seems to be updating the correct checksum.

**Comment by TarunParimi** [comment]
PR for the fix https://github.com/cloudera/yunikorn-k8shim/pull/16

**Comment by yangwwei** [comment]
Thanks @TarunParimi.
I tried the fix in #16, once applied, then I got the same checksum mismatch error. My go version is `go1.12.4 darwin/amd64`.
Since `go mod` is still under-development, does it make sense to update to a newer version of go and see if this issue goes away?

**Comment by TarunParimi** [comment]
Thanks @yangwwei . I upgraded to 1.12 version of go and this went away. I tried again with version 1.11.12 and the error didn't occur now after I did `go clean -modcache`.

I probably encountered this in the first place because I had the old checksum in my modcache. We can ignore this issue then as doing `go clean -modcache` seems to resolve it for me.

**Comment by yangwwei** [comment]
@TarunParimi , thank you.

Maybe we can add some doc in our README, when such checksum mismatch error shows up, we can suggest user to run `go clean -modcache`. Would you like to create a PR for this?

**Comment by TarunParimi** [comment]
@yangwwei . Sure. Will do it.

**Comment by TarunParimi** [comment]
@yangwwei Added a note in new PR https://github.com/cloudera/yunikorn-k8shim/pull/18 . Does this look good?

**Comment by yangwwei** [comment]
this is resolved by #18
